app/webpacker/css/admin_v3/components/buttons.scss

```diff
@@ -12,8 +12,8 @@ button:not(.plain):not(.trix-button),
   border: 1px solid $color-btn-bg;
   color: $color-btn-text;
   text-transform: uppercase;
-  line-height: 40px;
-  height: 40px;
+  line-height: $btn-regular-height - 2px; // remove 2px to compensate for border
+  height: $btn-regular-height;
```

Comment:
I guess the `line-height` is to ensure the text is vertically centred according to the desired `height`. I try to avoid this, because if the text ever wraps, it will be wrong. (Even though I'm sure it won't in this case).

For this reason I'd choose vertical padding, and try to get it right to reach the desired height. It actually needs to be considered during design. I think in the long run this strategy is more robust. 
But it's not a big thing, more just personal preference.

This can also be fiddly, because the end height is a result of several values, eg: `border-top + padding-top + font-size + padding-bottom + border-bottom`. 
Hmm.. so this way is fine ;)